examples/nl-core-SkinDisorder-01-Condition-01.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Condition xmlns="http://hl7.org/fhir">
+   <id value="nl-core-SkinDisorder-01-Condition-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Problem"/>
+   </meta>
+   <clinicalStatus>
+      <coding>
+         <system value="http://terminology.hl7.org/CodeSystem/condition-clinical"/>
+         <code value="active"/>
+         <display value="Active"/>
+      </coding>
+   </clinicalStatus>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="165232002"/>
```

Comment:
Terminology error: Terminology service failed while validating code '165232002' (system 'http://snomed.info/sct'): Cannot resolve canonical reference 'http://www.nanda.org/' to CodeSystem

I think we can ignore this